app/src/main/java/com/veriff/demo/login/LoginActivity.kt

```diff
@@ -0,0 +1,31 @@
+package com.veriff.demo.login
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import com.veriff.demo.MainActivity
+import com.veriff.demo.R
+import kotlinx.android.synthetic.main.activity_login.*
```

Comment:
:-1: I'm getting a compile error here, I guess synthetics aren't set up correctly? Personally I'm not a fan of synthetics - too much "magic" in my opinion. I know Jake Wharton is working on a better alternative but I have no idea when that is going to come out, probably later this year.

Reply:
Another option is to use data-binding but it will increase the build time. Heard they are working on incremental build for data-binding not sure if its out yet.

Reply:
Going back to findViewById()